diff --git generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.pbxproj generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.pbxproj
index 3c43063..86c7ee5 100644
--- generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.pbxproj	
+++ generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.pbxproj	
@@ -454,6 +454,7 @@
 				CODE_SIGN_ENTITLEMENTS = "OpenTelemetry Browser Ext Extension/OpenTelemetry_Browser_Ext_Extension.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = D38LM4UHT5;
 				ENABLE_HARDENED_RUNTIME = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OpenTelemetry Browser Ext Extension/Info.plist";
@@ -470,7 +471,7 @@
 					"-framework",
 					SafariServices,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Ext.Extension";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Extension.Extension";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
 				SWIFT_EMIT_LOC_STRINGS = YES;
@@ -484,6 +485,7 @@
 				CODE_SIGN_ENTITLEMENTS = "OpenTelemetry Browser Ext Extension/OpenTelemetry_Browser_Ext_Extension.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = D38LM4UHT5;
 				ENABLE_HARDENED_RUNTIME = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OpenTelemetry Browser Ext Extension/Info.plist";
@@ -500,7 +502,7 @@
 					"-framework",
 					SafariServices,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Ext.Extension";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Extension.Extension";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
 				SWIFT_EMIT_LOC_STRINGS = YES;
@@ -515,13 +517,16 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = "OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.entitlements";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = D38LM4UHT5;
 				ENABLE_HARDENED_RUNTIME = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OpenTelemetry Browser Ext/Info.plist";
 				INFOPLIST_KEY_CFBundleDisplayName = "OpenTelemetry Browser Ext";
+				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.developer-tools";
 				INFOPLIST_KEY_NSHumanReadableCopyright = "";
 				INFOPLIST_KEY_NSMainStoryboardFile = Main;
 				INFOPLIST_KEY_NSPrincipalClass = NSApplication;
@@ -537,7 +542,7 @@
 					"-framework",
 					WebKit,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Ext";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Extension";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_EMIT_LOC_STRINGS = YES;
 				SWIFT_VERSION = 5.0;
@@ -551,13 +556,16 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = "OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.entitlements";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = D38LM4UHT5;
 				ENABLE_HARDENED_RUNTIME = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OpenTelemetry Browser Ext/Info.plist";
 				INFOPLIST_KEY_CFBundleDisplayName = "OpenTelemetry Browser Ext";
+				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.developer-tools";
 				INFOPLIST_KEY_NSHumanReadableCopyright = "";
 				INFOPLIST_KEY_NSMainStoryboardFile = Main;
 				INFOPLIST_KEY_NSPrincipalClass = NSApplication;
@@ -573,7 +581,7 @@
 					"-framework",
 					WebKit,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Ext";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.theo.OpenTelemetry-Browser-Extension";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_EMIT_LOC_STRINGS = YES;
 				SWIFT_VERSION = 5.0;
diff --git generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.xcworkspace/xcuserdata/theodorebrockman.xcuserdatad/UserInterfaceState.xcuserstate generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.xcworkspace/xcuserdata/theodorebrockman.xcuserdatad/UserInterfaceState.xcuserstate
index 3934570..f3cd9a9 100644
Binary files generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.xcworkspace/xcuserdata/theodorebrockman.xcuserdatad/UserInterfaceState.xcuserstate and generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/project.xcworkspace/xcuserdata/theodorebrockman.xcuserdatad/UserInterfaceState.xcuserstate differ
diff --git generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/xcuserdata/theodorebrockman.xcuserdatad/xcschemes/xcschememanagement.plist generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/xcuserdata/theodorebrockman.xcuserdatad/xcschemes/xcschememanagement.plist
index de152ae..fc8acd1 100644
--- generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/xcuserdata/theodorebrockman.xcuserdatad/xcschemes/xcschememanagement.plist	
+++ generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext.xcodeproj/xcuserdata/theodorebrockman.xcuserdatad/xcschemes/xcschememanagement.plist	
@@ -10,5 +10,13 @@
 			<integer>0</integer>
 		</dict>
 	</dict>
+	<key>SuppressBuildableAutocreation</key>
+	<dict>
+		<key>6609A1FD2C0C9E59001FBE71</key>
+		<dict>
+			<key>primary</key>
+			<true/>
+		</dict>
+	</dict>
 </dict>
 </plist>
diff --git generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext/ViewController.swift generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext/ViewController.swift
index 54d952b..c33ee1f 100644
--- generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext/ViewController.swift	
+++ generated/safari-xcode/OpenTelemetry Browser Ext/OpenTelemetry Browser Ext/ViewController.swift	
@@ -9,7 +9,7 @@ import Cocoa
 import SafariServices
 import WebKit
 
-let extensionBundleIdentifier = "com.theo.OpenTelemetry-Browser-Ext.Extension"
+let extensionBundleIdentifier = "com.theo.OpenTelemetry-Browser-Extension.Extension"
 
 class ViewController: NSViewController, WKNavigationDelegate, WKScriptMessageHandler {
 
